### ISSUE TITLE
general "Dave's tasks" holdall (temporary storage, to be put in proper issues later)

### ISSUE THREAD

**Issue by ApplicationInsight** [AppertaFoundation/Diadem#67]
- [ ] move the 'Location' screen to appear before 'Patient"
- [ ] add a postCode field to the 'Location' screen
- [ ] add postCode data to a patient's demographic record on registering
- [ ] add a 'swipe-up' GUI panel to (almost) all screens (include 'float-over-everything' fakery)
- [ ] display items such as: 
  - [ ] link to context-sensitive help, save & quit option, feedback via userVoice, context s

**Comment by ColinSloane** [comment]
Very comprehensive list Dave.  We should go through it on a routine call sometime and see what we can prioritise and perhaps add more milestones to deal with some of them.

**Comment by ApplicationInsight** [comment]
@ColinSloane yes quite agree - some of items in the list will be pretty obvious (i.e. fix a bug), some will be pretty obvious but may need something checked, and some will be tricky / controversial and will need more discussion ... we can use Slack for many of them and Skype when needed :)
